.github/actions/configure-x64/action.yml

```diff
@@ -73,8 +73,6 @@ runs:
           ${{ inputs.skipSlow == 'false' && '--with-snmp' || '' }} \
           ${{ inputs.skipSlow == 'false' && '--with-unixODBC' || '' }} \
           ${{ inputs.skipSlow == 'false' && '--with-pdo-odbc=unixODBC,/usr' || '' }} \
-          $([ -d "/opt/oracle/instantclient" ] && echo '--with-pdo-oci=shared,instantclient,/opt/oracle/instantclient') \
```

Comment:
The CI should be moved to the PECL repos.

Reply:
I'll add you as a collaborator so you can do so.

Reply:
Thank you, I contribute to CI a lot, but I am not a C expert so I would be happy if someone else can do this.

The first step should be to convert GH workflows into resuable GH actions so the PECL repos can take the advantage of sanitizers, nightly jobs, ...

Also CI parts like https://github.com/php/php-src/blob/php-8.3.2/.github/workflows/nightly.yml#L81 should be moved.

To the migration, I wonder if monorepo for all pecl packages under php org was considered. IMO, managing oci8 and pdo_oci separately imply a lot of extra time and communication needed to maintain. Not mentioning keeping the changes consistent is quite resource demanding.